### PR TITLE
deps: upgrade design-system components

### PR DESCRIPTION
**Pull request by alantoa**
# Why

- root package has `react-native-tab-view-next` package.
- current bit.could and npm versions are inconsistent.
- lightbox not working and relying on `photoswipe` CSS is not good.
- `Image` component has many incorrect types, and props do not support width/height/borderRadius.
- `staging` branch build error and `sb-react` has some issues.

# How
- move the root package to the expo folder.
- upgrade bit and npm versions.
- fix the lightbox component and remove `photoswipe`, now only support Native, the web will add to the next PR.
- improve `Image` components, fixed types and clear code.
- merge `staging` branch and fixed `staging` issues.

# Test Plan
- reinstall packages and check the project if it's working.
- check lightbox if it's work on Native.

### CODE REVIEW

**Review by enesozturk — APPROVED** [review]

**Comment by enesozturk** [comment]
Ops it's draft, sorry

File: apps/storybook-react/.storybook/preview.js [hunk 1]
```diff
@@ -10,8 +10,7 @@ import { ToastProvider } from "@showtime-xyz/universal.toast";
 import { View } from "@showtime-xyz/universal.view";
 
 import { linking } from "app/navigation/linking";
-
-import { theme } from "design-system/theme";
+import { ThemeProvider } from "app/providers/theme-provider";
```
Comment (intergalacticspacehighway) [inline apps/storybook-react/.storybook/preview.js:13]: 💯

File: packages/design-system/color-scheme/color-scheme.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import * as React from "react";
+import React, { createContext, useContext } from "react";
 
 type ColorScheme = "light" | "dark" | null | undefined;
 
 type SetColorScheme = (colorScheme: ColorScheme) => void;
 
-export const ColorSchemeContext = React.createContext(
+export const ColorSchemeContext = createContext(
```
Comment (intergalacticspacehighway) [inline packages/design-system/color-scheme/color-scheme.tsx:7]: nice, i am not the only one with ocd 💯 😌

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️ 🙏

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Pull request merged**